code-style.md

```diff
@@ -55,6 +55,12 @@ To configure IntelliJ/Android Studio according to this style guide, you can eith
 * * Remove `import java.util.*` entry by clicking the `-` button
 <sup>[[link](#setup-imports)]</sup>
 
+### Set empty line at the end of a file
+
+* Go to `Settings` | `Editor` | `General`
+* Scroll to the `Save Files` section
```

Comment:
It's in `Other` section btw

Reply:
Wrong:
![Capture d’écran 2020-10-16 à 15 46 21](https://user-images.githubusercontent.com/8343416/96266339-d5cf3580-0fc6-11eb-9d40-08c963c43595.png)

You should update to Android Studio 4.1 if it's not the case.